braid/braid.c

```diff
@@ -509,24 +508,54 @@ braid_Destroy(braid_Core  core)
       _braid_TFree(_braid_CoreElt(core, tnorm_a));
       _braid_TFree(_braid_CoreElt(core, rdtvalues));
 
-      /* Destroy stored Lyapunov exponents */
-      if (_braid_CoreElt(core, lyap_exp))
+      if (grids[0] != NULL)
```

Comment:
This diff is a bit of a mess, but all I did was add this check for `grids[0] != NULL` to prevent a segfault in case the Braid hierarchy was never set up, which was happening occasionally while developing the Julia interface.